src/main/java/tsundere/Tsundere.java

```diff
@@ -0,0 +1,22 @@
+package tsundere;
+
+import tsundere.task.Storage;
+
+
+import javafx.application.Application;
```

Comment:
Consider running the Java check-style.

The imports should be like this:

package tsundere;

import javafx.application.Application;
import tsundere.task.Storage;